docs/Quickstart.md

```diff
@@ -38,13 +61,13 @@ var result = chunk.TargetFunc();
 rock.Verify();
 ```
 
-Note that all mocks generated with Rocks are strict. That is, if you didn't set up an expectation for the `TargetAction()` call, the `Verify()` call would fail.
+Note that all mocks generated with Rocks are **strict**. That is, if you didn't set up an expectation for the `TargetAction()` call, the `Verify()` call <ins>would fail.</ins>
```

Comment:
Don't need the emphasis on the words "strict" or "would fail."

Reply:
Acknowledged, I'll remove them and commit the changes together. Thanks for the feedback!